src/services/pagoPAClientFactory.ts

```diff
@@ -18,10 +19,15 @@ export default class PagoPAClientFactory
     codiceContestoPagamento: string,
     rptIdFromString: string
   ): ProxyPagoPA {
+    // Avoid to retry to send the request if proxy reply with a 500 HTTP code
```

Comment:
I looks more like that with this option the client will NEVER retry a request, even for a timeout, is that the case?

Reply:
As defined here:
https://github.com/Azure/ms-rest-js/blob/master/lib/serviceClient.ts
"if set to true, turn off the default retry policy."

I'm going to update the comment there.